go.mod

```diff
@@ -1,11 +1,12 @@
+module git.paas.vn/BizFly-PaaS-Cloud/bizfly-cloud-controller-manager
```

Comment:
Use github.com instead @cuonglm

Reply:
https://pkg.go.dev/search?q=git.paas.vn%2FBizFly-PaaS-Cloud%2Fbizfly-cloud-controller-manager

Luckily, there's no usage of the module yet.

Reply:
cc @cpfriend1721994

Sorry for hjacking your PR here, as now we realize that no one depend on the module yet, so we can change the name.

Also sorry if @greatbn closes your PR without any notice.